index.php

```diff
@@ -63,9 +63,10 @@
 	$hyphaQuery = $hyphaRequest->getRelativeUrlPath();
 	$hyphaUrl = $hyphaRequest->getRootUrl();
 
+	$pathToTheme = 'data/themes/' . Hypha::$data->theme;
+
 	// Shortcut for direct file requests
-	if (strpos($hyphaQuery, 'data/themes/default/') === 0) serveFile($hyphaQuery, false);
-	if (strpos($hyphaQuery, 'data/fonts/') === 0) serveFile($hyphaQuery, false);
```

Comment:
If fount are outside of the theme, it will be more difficult to transfer a theme from one installation to another.

Althought this may lead to some duplification of font files, I think the ease of installing would win here.